tests/e2e/_test_es_dump.py

```diff
@@ -1,31 +1,31 @@
 import pathlib
```

Comment:
For this to work, I guess, we need to make sure the entire infrastructure is in place. 

It is good you exclude these tests from the regular runs by prefixing them with "_"